services/core-web/src/components/noticeOfWork/applications/manageDocuments/ManageDocumentsTab.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { getFormValues } from "redux-form";
+import {
+  updateNoticeOfWorkApplication,
+  fetchImportedNoticeOfWorkApplication,
+  fetchNoticeOfWorkApplicationReviews,
+} from "@common/actionCreators/noticeOfWorkActionCreator";
+import {
+  getNoticeOfWork,
+  getImportNowSubmissionDocumentsJob,
+  getNoticeOfWorkReviews,
+} from "@common/selectors/noticeOfWorkSelectors";
+import { getGeneratableNoticeOfWorkApplicationDocumentTypeOptions } from "@common/selectors/staticContentSelectors";
+import { getDropdownInspectors } from "@common/selectors/partiesSelectors";
+import CustomPropTypes from "@/customPropTypes";
+import * as FORM from "@/constants/forms";
+import NOWSideMenu from "@/components/noticeOfWork/applications/NOWSideMenu";
+import NOWTabHeader from "@/components/noticeOfWork/applications/NOWTabHeader";
+import NOWApplicationManageDocuments from "@/components/noticeOfWork/applications/manageDocuments/NOWApplicationManageDocuments";
+
+/**
+ * @class ManageDocumentsTab- contains all information relating to the Securities/Bond tracking on a Notice of Work Application.
+ */
+
+const propTypes = {
+  noticeOfWork: CustomPropTypes.importedNOWApplication.isRequired,
+  updateNoticeOfWorkApplication: PropTypes.func.isRequired,
+  fetchImportedNoticeOfWorkApplication: PropTypes.func.isRequired,
+  fetchNoticeOfWorkApplicationReviews: PropTypes.func.isRequired,
+  fixedTop: PropTypes.bool.isRequired,
+  importNowSubmissionDocumentsJob: PropTypes.bool.isRequired,
+  inspectors: CustomPropTypes.groupOptions.isRequired,
+  formValues: CustomPropTypes.importedNOWApplication.isRequired,
+  noticeOfWorkReviews: PropTypes.arrayOf(CustomPropTypes.NOWApplicationReview).isRequired,
+};
+
+export class ManageDocumentsTab extends Component {
+  state = {
+    isInspectorsLoaded: true,
+  };
+
+  componentDidMount = () =>
+    this.props.fetchNoticeOfWorkApplicationReviews(this.props.noticeOfWork.now_application_guid);
+
+  handleUpdateInspectors = (values, finalAction) => {
+    this.setState({ isInspectorsLoaded: false });
```

Comment:
Is this needed on the manage documents tab? Where are inspectors being updated on this tab?

Reply:
I think this was a copy paste by mehkti cause he used the administrative tab as his base. I am removing it.